runtime/lua/vim/lsp/client.lua

```diff
@@ -1082,27 +1082,11 @@ function Client:supports_method(method, bufnr)
   return false
 end
 
---- Get options for a method that is registered dynamically.
---- @param method string
---- @param bufnr? integer
---- @return lsp.LSPAny?
-function Client:_get_registration_options(method, bufnr)
-  if not self:_supports_registration(method) then
-    return
-  end
-
-  local reg = self:_get_registration(method, bufnr)
-
-  if reg then
-    return reg.registerOptions
-  end
-end
-
```

Comment:
👀

Reply:
Not used anywhere, must be an orphan after a deprecation was removed.